server/src/main/java/org/apache/druid/indexing/overlord/IndexerMetadataStorageCoordinator.java

```diff
@@ -179,6 +179,12 @@ List<DataSegment> retrieveUnusedSegmentsForInterval(
       @Nullable DateTime maxUsedStatusLastUpdatedTime
   );
 
+  List<DataSegment> retrieveUnusedSegmentsForExactIntervalAndVersion(
```

Comment:
This does not need to be an interface method.

Reply:
Done.